app/assets/stylesheets/admin/base.scss

```diff
@@ -0,0 +1,12 @@
+@import "../shared/color";
+@import "../shared/utilities";
+@import "../shared/markdown";
+@import "form";
+
+hr {
+  border: 0;
+  height: 0;
```

Comment:
Properties should be ordered border, border-bottom, height, margin-bottom, padding-top